docs/demo/bundle.ts

```diff
@@ -3,33 +3,4 @@ export * from '@patternfly/pfe-icon';
 import 'html-include-element';
 import 'api-viewer-element';
 
-import '@patternfly/pfe-band';
-import '@patternfly/pfe-card';
-
-import '@patternfly/pfe-jump-links';
-import '@patternfly/pfe-modal';
-import '@patternfly/pfe-tabs';
-import '@patternfly/pfe-collapse';
-import '@patternfly/pfe-accordion';
-import '@patternfly/pfe-icon-panel';
-import '@patternfly/pfe-primary-detail';
-
-import '@patternfly/pfe-autocomplete';
-import '@patternfly/pfe-avatar';
-import '@patternfly/pfe-badge';
-import '@patternfly/pfe-button';
-import '@patternfly/pfe-clipboard';
-import '@patternfly/pfe-codeblock';
-import '@patternfly/pfe-cta';
-import '@patternfly/pfe-datetime';
-import '@patternfly/pfe-dropdown';
-import '@patternfly/pfe-health-index';
-import '@patternfly/pfe-icon';
-import '@patternfly/pfe-markdown';
-import '@patternfly/pfe-number';
-import '@patternfly/pfe-page-status';
-import '@patternfly/pfe-progress-indicator';
-import '@patternfly/pfe-progress-steps';
-import '@patternfly/pfe-readtime';
-import '@patternfly/pfe-select';
-import '@patternfly/pfe-toast';
+import './components.js';
```

Comment:
What generates this file?

Reply:
https://github.com/patternfly/patternfly-elements/blob/bf3224be19303b2a8d5b21ed66a9f1c3a064b0cf/tools/pfe-tools/esbuild.ts#L107-L126

Previously, we just wrote a new line to this file every time the generator ran,

but now, we just generate that file from the `/elements` dir anytime we need to.

Reply:
noice!